ape_etherscan/explorer.py

```diff
@@ -47,23 +50,21 @@ def _client_factory(self) -> ClientFactory:
             )
         )
 
-    def get_contract_type(self, address: AddressType) -> Optional[ContractType]:
+    def get_source(self, address: AddressType) -> Source:
+        code = self._get_source_code(address)
+        return Source(content=code.source_code)
```

Comment:
seems like this only works for "flattened" sources?

If this can be a higher level `.get_manifest` function, this can encapsulate all the `Sources` in one manifest together, including `Compiler` settings as well (and perhaps even `ContractType` entry for use with `.get_contract_type`)

Reply:
Ah, let me check out what a JSON verified one looks like.
For some reason I thought Etherscan still show the flattened-sources even for those but maybe I was wrong

Reply:
OK did this, should work with the JSON approach now as well!